cmd/cbnt-prov/cmd.go

```diff
@@ -382,19 +382,19 @@ func (g *generateKMCmd) Run(ctx *context) error {
 }
 
 func (g *generateBPMCmd) Run(ctx *context) error {
-	var options *bg.BootGuardOptions
+	var options *cbnt.BootGuardOptions
 	if g.Config != "" {
-		bgo, err := bg.ParseConfig(g.Config)
+		cbnto, err := cbnt.ParseConfig(g.Config)
 		if err != nil {
 			return err
 		}
-		options = bgo
+		options = cbnto
 	} else {
-		var bgo bg.BootGuardOptions
-		bgo.BootPolicyManifest.BPMH.BPMRevision = g.Revision
-		bgo.BootPolicyManifest.BPMH.BPMSVN = g.SVN
-		bgo.BootPolicyManifest.BPMH.ACMSVNAuth = g.ACMSVN
-		bgo.BootPolicyManifest.BPMH.NEMDataStack = g.NEMS
+		var cbnto cbnt.BootGuardOptions
+		cbnto.BootPolicyManifest.BPMH.BPMRevision = g.Revision
```

Comment:
cbnto?

Reply:
Ah, thanks. Should rename BootGuardOptions as well.

Reply:
Renamed it to Options. Access now via cbnt.Options